doc/Command-Reference.md

```diff
@@ -1371,6 +1371,33 @@ When the optional argument "max_priority"  is specified, each rule’s priority
 
 Go Back To [Beginning of the document](#) or [Beginning of this section](#acl)
 
+**config acl add table**
+
+This command is used to create new ACL tables.
+
+- Usage:
+  ```
+  config acl add table [OPTIONS] <table_name> <table_type> [-d <description>] [-p <ports>] [-s (ingress | egress)] [--expand_vlans]
```

Comment:
>--expand_vlans [](start = 116, length = 14)

`expand_vlans` should be an implementation detail, not a CLI option. User could not specify this option freely in current image.
This is similar to minigraph.py.

Reply:
Actually, I guess we don't have that option available in orchagent at this time, so it might be better to handle this internally and document the behavior in the CLI. Thoughts?

Reply:
Agree with "handle this internally and document the behavior in the command reference".